docs/assignment1.md

```diff
@@ -1,62 +1,154 @@
+world and other players. These games have evolved from simple text-based games to more complex artistic games of today. Going back to the classics, this project will be a minimal version of Zork, which takes place in a post-apocalyptic world.
+Our protagonist wakes up in the middle of the day, seeing an emergency alert on his phone of an imminent missile strike. They go on a mission to find their
+family or risks dying in the nuclear strike.
+
+The minimal game of Zork consists of a protagonist, a virtual world, and objects and virtual characters. We took inspiration from the Black Isle Studios
+game [Fallout 2](https://store.steampowered.com/app/38410/Fallout_2_A_Post_Nuclear_Role_Playing_Game/) with the post-apocalyptic world using simple objects as currency. In the *Fallout* game series, bottle caps are used as currency. In our game,
+sporks are used as the currency (the word rhymes with Zork). Post-apocalyptic genre games became incredibly popular in the mid-2000s to the early 2010s, so it is difficult to
+point to a specific game that inspired that world in our game.
+
+The project track specifies that our game be loyal to the original Zork, which will run completely in the terminal. The termainal with have color-coded text, in order
+to indicate to the player whether text describes a player, a situation, or the protagonists' health, etc. As an extension, we can create an alternative version
+of the game that runs with a GUI, to mimic the second-generation text-based games. A "GUI" can even be included within the console that is used to play the game, by using Java libraries such as Text-IO.
+With games such as Zork and various other text-based adventure games, the target audience is teenagers and young adults, but as this game will not
+include anything incredibly explicit and violent like the *Fallout* game series, this game may be played at an age.
+
+This gameplay will be achieved by creating models of the two important factors of the game: the scenes in which the character is in at a given moment,
+and the characters that the player encounters in the scenes. Each of the scenes will be based on the same template, but with different objects, characters
+and places to explore. Since we are working with object-oriented programming, a separate Scenes model will be created, with a different scene being created
+at points in the game with specific parameters, with some possibly randomized. Each scene will have different options but falling under one of the three
+umbrellas of movement, exploration or interaction. The interaction can be with characters, objects. Interactions can also be of different types, like picking
+up an object or walking in a certain direction. These interactions help the player to reach the goal or to fail and lose the game.
+
+Word Count: 552
 
```

Comment:
Cool, I like your ideas and creativity. Make sure though, that your emphasis on story is not heavily tied to functionality since the game must be independent of the storyline.